MainDemo.Wpf/Buttons.xaml

```diff
@@ -45,335 +46,596 @@
             <RowDefinition Height="Auto" />
             <RowDefinition Height="Auto" />
             <RowDefinition Height="Auto" />
-        </Grid.RowDefinitions>        
-        <TextBlock
+        </Grid.RowDefinitions>
+        <DockPanel LastChildFill="False">
+            <TextBlock
             Style="{StaticResource MaterialDesignHeadline5TextBlock}"
             Text="Buttons"/>
+            <ToggleButton x:Name="ButtonsEnabled"
```

Comment:
I really like the addition of this! I do wonder if more areas could benefit from this, and if so if it would be good to move it up under the menu in the top right next to the dark/light theme toggle.